src/siotls/examples/minih1s.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+"""
+Mini HTTP/1.1 server over TLS with openssl, used to test the siotls
+client against another TLS implementation.
+"""
+
+__file_path__ = __import__('pathlib').Path(__file__)  # eventually standard??
+
+import logging
+import socket
+import ssl
+import textwrap
+import sslkeylog
+
+HOST = '127.0.0.1'
+PORT = 8446
+
+logging.basicConfig(level='INFO')
+logger = logging.getLogger(__file_path__.stem)
+
+sslkeylog.set_keylog('/home/julien/.tlskeylogfile')
+context = ssl.SSLContext(ssl.PROTOCOL_TLS_SERVER)
+context.set_alpn_protocols(['http/1.1'])
+context.load_cert_chain(
+    __file_path__.joinpath('../../data/self-signed-cert.pem').resolve(),
+    __file_path__.joinpath('../../data/self-signed-key.pem').resolve(),
+)
+
+with socket.socket(socket.AF_INET, socket.SOCK_STREAM, 0) as sock:
+    sock.bind((HOST, PORT))
+    sock.listen(5)
```

Comment:
backlog parameter is optional since 3.5 you old man
```suggestion
    sock.listen()
```